05-UI-ReactJS/react-builder-master/source/index.js

```diff
@@ -41,33 +41,35 @@ class Application extends React.Component{
   }
 
   getMovies () {
-     let movies = localStorage.getItem("movies") || [];
-     return JSON.parse(movies);
-   }
-
-  editMovie(){
+    if (localStorage.getItem("movies") === null){
+    let movies = [];
```

Comment:
declare and initialize *movies* to an empty array outside of the if block, and assign it  a value if localStorage is setted, something like:
````
let movies = [];

if (localStorage.getItem('movies')) {
    movies = localStorage.getItem('movies')
}

return movies;

````